packages/app/src/components/nominationBanner/nominationBanner.js

```diff
@@ -52,10 +60,21 @@ const NominationBanner = ({ nomination }) => {
               </span>
             </div>
             <div className="column name">
-              <button className=" decline-button" onClick={declineApplication} style={{ background: '#f72314', color: '#ffffff', border: '#929292', fontWeight: 'bold' }}>
+              <button className=" decline-button" onClick={toggleModalState} style={{ background: '#f72314', color: '#ffffff', border: '#929292', fontWeight: 'bold' }}>
                 Decline Application
               </button>
             </div>
+            <div className={`modal-background modalShowing-${modalState}`}>
+              <div className="modal-container">
+                <button className= "exit-button" onClick={toggleModalState} >&times;</button>
+                
+                <h3 className="modal-text">Do you want to decline the application?</h3>
+                <div className="modal-buttons">
+                  <button className="button-yes" onClick={()=>{declineApplication(); toggleModalState()}}>Yes</button>
```

Comment:
The calls to `declineApplication` and `toggleModalState` should probably be on two separate lines. Generally best to have only one statement per line for readability purposes.

Reply:
Done!